lib/chef/resource/registry_key.rb

```diff
@@ -67,6 +67,8 @@ def initialize(name, run_context = nil)
         @values, @unscrubbed_values = [], []
       end
 
+      property :sensitive, [ TrueClass, FalseClass ], default: false, coerce: proc { |x| password ? true : x }
```

Comment:
Just to follow up from Slack since I'm now on a real keyboard, this is a weird thing to add. The `sensitive` property is supported on all resources already, and the coerce doesn't actually help with masking passwords in most cases. I think you mean to mark the `password` property sensitive instead?

Reply:
yeah we also ran into cases where people want to pass a password and want sensitive off for debugging or something like that i think.